tests/target/configs/format_cargo_toml/true/array_values/Cargo.toml

```
@@ -0,0 +1,70 @@
+# rustfmt-format_cargo_toml: true
+[package]
+name = "std"
+version = "0.0.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rust-lang/rust.git"
+description = "The Rust Standard Library"
+
+[lib]
+crate-type = ["dylib", "rlib"]
+
+[dependencies]
+alloc = { path = "../alloc", public = true }
+cfg-if = { version = "1.0", features = ["rustc-dep-of-std"] }
+core = { path = "../core", public = true }
+panic_abort = { path = "../panic_abort" }
+panic_unwind = { path = "../panic_unwind", optional = true }
+
+[dependencies.libc]
+version = "0.2.148"
+default-features = false
+features = ["rustc-dep-of-std"]
+public = true
+
+[dependencies.reqwest]
+version = "0.11"
+features = ["blocking", "rustls-tls", "rustls-tls-native-roots"]
+optional = true
+default-features = false
+
+[dependencies.std_detect]
+path = "../stdarch/crates/std_detect"
+default-features = false
+features = ["rustc-dep-of-std"]
+
+[target.'cfg(not(all(windows, target_env = "msvc", not(target_vendor = "uwp"))))' .dependencies]
+miniz_oxide = { version = "0.7.0", optional = true, default-features = false }
+addr2line = { version = "0.21.0", optional = true, default-features = false }
```

Should be sorted alphabetically
```suggestion
addr2line = { version = "0.21.0", optional = true, default-features = false }
miniz_oxide = { version = "0.7.0", optional = true, default-features = false }
```